### PR TITLE
Octicons 3.0

### PR DESCRIPTION
**Pull request by aaronshekey**
This is the biggie. As tracked in CHANGELOG this updates the following:

### Removes

- `microscope`
- `beer`
- `split`
- `puzzle`
- `steps`
- `podium`
- `timer`
- all `alignment` icons
- all `move` icons
- all `playback` icons
- all `jump` icons

### Adds

- `beaker`
- `bell`
- `desktop-download`
- `watch`

### Line-weight changes, sizing normalization, and new drawings

- `circle-slash`
- `lock`
- `cloud-upload`
- `cloud-download`
- `plus`
- `✕`
- `broadcast`
- `lock`
- all `repo` icons
- organization
- person
- all `chevrons` & `triangles`
- all `diff` icons
- `clippy`
- all `issue` and circular icons
- `rss`
- `ruby`
- `cancel`
- `settings`
- `mirror`
- `external-link`
- `history`
- `gear`
- `settings`
- `info`
- `history`
- `package`
- `gist-secret`
- `rocket`
- `law`
- `telescope`
- `search`
- `tag`
- `normal-screen`
- `iphone`
- `no-new-line`
- `desktop`
- all `git` icons
- `circuit-board`
- `heart`
- `home`
- `briefcase`
- `wiki`
- `bookmark`
- `briefcase`
- `calendar`
- `color-mode`
- `comment`
- `discussions`
- `credit-card`
- `dashboard`
- `camera`
- `video`
- `bug`
- `desktop`
- `ellipses`
- `eye`
- all `files` & `folders`
- `fold`
- `unfold`
- `gift`
- `graph`
- `hubot`
- `inbox`
- `jersey`
- `keyboard`
- `light-bulb`
- `link`
- `location`
- `mail`
- `mail-read`
- `marker`
- `plug`
- `mute`
- `pencil`
- `push-pin`
- `fullscreen`
- `unfullscreen`
- `server`
- `sign-in`
- `sign-out`
- `tag`
- `terminal`
- `thumbs-up`
- `thumbs-down`
- `trash`
- `unmute`
- `versions`
- `gist`
- `key`
- `megaphone`
- `checklist`

### CODE REVIEW

**Comment by cameronmcefee** [comment]
:cool: Let's let this sit while the docs team gets things updated on their end.

**Comment by aaronshekey** [comment]
Okie dokie, the docs are ready. I'm gonna merge this once I get your thumbs up.

File: octicons/octicons.less [hunk 1]
```diff
@@ -39,7 +36,9 @@
 .octicon-arrow-small-right:before { content: '\f071'} /*  */
 .octicon-arrow-small-up:before { content: '\f09f'} /*  */
 .octicon-arrow-up:before { content: '\f03d'} /*  */
-.octicon-beer:before { content: '\f069'} /*  */
+.octicon-microscope:before,
```
Comment (zerowidth) [inline octicons/octicons.less]: if microscope is going away, should this line be here?

File: octicons/sprockets-octicons.scss [hunk 2]
```diff
@@ -36,7 +33,9 @@
 .octicon-arrow-small-right:before { content: '\f071'} /*  */
 .octicon-arrow-small-up:before { content: '\f09f'} /*  */
 .octicon-arrow-up:before { content: '\f03d'} /*  */
-.octicon-beer:before { content: '\f069'} /*  */
+.octicon-microscope:before,
```
Comment (zerowidth) [inline octicons/sprockets-octicons.scss]: Same re: :microscope:

File: octicons/octicons.less (hunk 1 above)
Comment (cameronmcefee) [inline octicons/octicons.less]: This is an alias to prevent breakage

Comment (aaronshekey) [inline octicons/octicons.less]: I've left the microscope as an alias for the beaker for now to help migrate things. We can eventually remove that if it turns out it isn't needed.

**Comment by cameronmcefee** [comment]
@aaronshekey I noticed I forgot to add the `.scss` template to the generator, which ended up deleting the file here. I added it back in so this should be good to go. :+1:

**Comment by aaronshekey** [comment]
@cameronmcefee Good looking out!

**Comment by xt0rted** [comment]
When 3.0.1 was released it was tagged as `v3.0.1` yet the link in my notifications says `3.0.1` and points to `https://github.com/github/octicons/releases/tag/3.0.1` which returns a 404.

**Comment by aaronshekey** [comment]
@xt0rted yeah, I messed up the naming convention on the tags. https://github.com/github/octicons/releases/tag/v3.0.1 is the proper latest release. My bad!

**Comment by XhmikosR** [comment]
Hey, guys. I was wondering, is there any replacement for the removed podium icon?

**Pull request merged**